mono/metadata/cominterop.c

```diff
@@ -3681,7 +3635,7 @@ ves_icall_System_Runtime_InteropServices_Marshal_PtrToStringBSTR (gpointer ptr)
 gpointer
 ves_icall_System_Runtime_InteropServices_Marshal_StringToBSTR (MonoString* ptr)
 {
-	return mono_string_to_bstr(ptr);
+	return mono_string_to_bstr (ptr);
```

Comment:
This is silly. why do we bother doing `register_icall (mono_string_to_bstr, ...)` when we already have a perfectly good icall that does exactly the same thing.

It would be nice if we could make `mono_string_to_bstr` just a utility function and replace uses of it as an icall by this one.

Is there some `#ifdef` logic that i'm missing why that's not possible?